packages/lesmis/src/api/queries/useVitalsData.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import { useQuery } from 'react-query';
+import { get } from '../api';
+import { useEntityData } from './useEntityData';
+import { useEntitiesData } from './useEntitiesData';
+
+const getDecendantsOfType = (entities, rootEntityCode, type) => {
```

Comment:
I would have expected that this kind of aggregation would happen in the report building? Do you know if that's possible?

Reply:
Not currently, gonna be talking to Rohan about that later today.

Reply:
Entity aggregation in the report server is coming down the line, but for MVP we're stuck doing it on the front end